### PR TITLE
Landlord immigration check - Implement non-EEA family member of someone from the EU, EEA or Switzerland

### PR DESCRIPTION
**Pull request by leenagupte**
Trello card: https://trello.com/c/7uh3hVfS

Follows on from the changes in branch: [feature/landlord-immigration-check-flow-changes](https://github.com/alphagov/smart-answers/compare/feature/landlord-immigration-check-flow-changes)

## Factcheck
[landlord-immigration-check](https://smart-answers-pr-2960.herokuapp.com/landlord-immigration-check/y/B1%201PW/yes/yes)

## Expected changes
* [URL on GOV.UK](https://www.gov.uk/landlord-immigration-check/y/B1%201PW/yes/yes)
   * Add the flow for tenants who are a non-EEA family member of a British or European citizen.

### Before
screen shot 2017-03-13 at 17 24 55

### After
screen shot 2017-03-13 at 17 26 08

### CODE REVIEW

File: test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/yes.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
 
-This person can rent the property.
+This person can currently rent the property.
 
-You need to make a copy of the [relevant parts of the documents](/check-tenant-right-to-rent-documents) and keep the copies for 12 months after the tenancy has ended.
+You need to make a copy of the [relevant parts of the documents](/check-tenant-right-to-rent-documents).
 
-You don’t have to make this check again.
+If the tenant's right to stay in the UK is for less than 12 months, check again just before 12 months' time. Otherwise check again just before their right to stay in the UK runs out.
```
Comment (ikennaokpala) [inline test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/yes.txt]: Need to verify that the text in outcome ok 7 is correct

File: test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+
+You must [check with the Home Office](https://eforms.homeoffice.gov.uk/outreach/lcs-application.ofml) to find out if the tenant is allowed to rent property. You’ll need the person’s Home Office reference number.
+
+You’ll get an answer within 2 working days.
+
+If the person’s allowed to rent, the Home Office will tell you if you need to check their documents again later.
```
Comment (leenagupte) [inline test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt]: @ikennaokpala I think that `No 2` is what will become `No 3` in the logic document. Polly has already agreed to update the logic document with the new template text.

File: test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/yes.txt (hunk 1 above)
Comment (leenagupte) [inline test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/yes.txt]: @ikennaokpala I have left a comment for Polly on the Trello card to double check this.

File: test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt (hunk 2 above)
Comment (leenagupte) [inline test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt]: @ikennaokpala Do you happen to know which commits are over the character limit, so I can correct them?

Comment (ikennaokpala) [inline test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt]: @leenagupte here d0f2b65e17b61d9345605e5237b6de8b999b3422 and 94bbd03c49f76c0aaaa8573b94144dc41c8fcd92

Comment (leenagupte) [inline test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt]: @ikennaokpala I have amended the commit titles for these two commits.

**Pull request merged**